packages/sdk/src/index.ts

```diff
@@ -1,3 +1,13 @@
+import { MetromApiClient } from "./client/backend";
+import { SERVICE_URLS } from "./commons";
+
+export const metromDevelopmentApiClient = new MetromApiClient(
```

Comment:
I honestly think it might be better to export this as:

```
export const METROM_API_CLIENT: Record<Environment, MetromApiClient> = ...
```

Reply:
Addressed in a823d357b01221c0c0abb3ed5651b3ed47663090.